src/NzbDrone.Core.Test/TvTests/ShouldRefreshSeriesFixture.cs

```diff
@@ -66,6 +71,8 @@ private void GivenRecentlyAired()
         [Test]
         public void should_return_true_if_running_series_last_refreshed_more_than_6_hours_ago()
         {
+            GivenSeriesLastRefreshedHalfADayAgo();
```

Comment:
This test passed as-is for our main builds, but fails for the PR builds (same tests are run, so its rather bizarre), can we drop the initial set of the last refreshed time/set it to UtcNow instead, that will make this method make more sense and I don't see a reason to nit at it since we do it for the other tests.